stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -12621,3 +12631,136 @@ fn write_signer_update(
         );
     }
 }
+
+/// Test SIP-031 activation
+///
+/// - check epoch 3.2 is active
+/// - check sip031 boot contract has a balance of 200_000_000 STX (TODO)
```

Comment:
This isn't in here yet - I'd just mark this as TODO
```suggestion
/// - TODO: check sip031 boot contract has a balance of 200_000_000 STX (TODO)
```